install_config/install/prerequisites.adoc

```diff
@@ -214,15 +214,115 @@ The following must be set up in your environment before {product-title} can be
 installed.
 
 [[prereq-dns]]
-
 === DNS
 
+A fully functional DNS environment is a requirement for {product-title} to work
+correctly. Adding entries into the *_/etc/hosts_* file is not enough, because
+the DNS resolver libraries do not query that file.
+
+To configure the {product-title} DNS environment:
+
+- xref:../../install_config/install/prerequisites.adoc#dns-config-prereq[Complete DNS configuration]
+- (Optionally) xref:../../install_config/install/prerequisites.adoc#wildcard-dns-prereq[configure a wildcard for the router]
+
+Key components of {product-title} run themselves inside of containers. By
+default, these containers receive their *_/etc/resolv.conf_* DNS configuration
+file from their host. {product-title} then inserts one DNS value into the pods
+(above the node's nameservers values). That value is defined in the
```

Comment:
Does "nameservers" need to be possessive? If not, I'm thinking it doesn't need the last s? Let me know if this doesn't make sense.